daemon/api_test.go

```diff
@@ -3665,11 +3665,28 @@ func snapList(rawSnaps interface{}) []map[string]interface{} {
 	return snaps
 }
 
+// inverseCaseMapper implements SnapMapper to use lower case internally and upper case externally.
+type inverseCaseMapper struct {
+	ifacestate.IdentityMapper // Embed the identity mapper to reuse empty state mapping functions.
+}
+
+func (m *inverseCaseMapper) RemapSnapFromRequest(snapName string) string {
```

Comment:
(nitpick^2) I wonder if we should have something like `from-req-$snapName` (and vice for the RemapSnapToResponse).